src/applications/sah/sahg/config/form.js

```diff
@@ -35,9 +22,9 @@ const {
   date,
   dateRange,
   usaPhone,
-  bankAccount,
+  // bankAccount,
   toursOfDuty,
-} = commonDefinitions;
+} = fullSchema.definitions;
```

Comment:
The schema for 26-4555 doesn't have a definition for `bankAccount`. This is the cause of the flaky test.

![image](https://user-images.githubusercontent.com/2008881/145492031-bd0e2e72-0ba2-4774-9238-50f5d0b25ec2.png)

https://github.com/department-of-veterans-affairs/vets-json-schema/blob/5b4775402bec413f6c55ee7288cdd05b1c048d37/dist/26-4555-schema.json#L6-L215